app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -4,9 +4,7 @@ class Admin::AdminSetsController < ApplicationController
     include Hyrax::CollectionsControllerBehavior
 
     before_action :authenticate_user!
-    before_action :ensure_manager!, except: [:show]
```

Comment:
Checks for manager and viewer are covered by cancan authentication.  Feature tests and controller tests behave correctly for each user type.  Interactions with the test app in the browser also confirm that the controller limits access as expected.